src/components/LibraryComparison/index.tsx

```diff
@@ -15,10 +15,10 @@ type LibraryNode = {
 
 type LibraryFeatures = {
     eventCapture: boolean
-    autoCapture: boolean
```

Comment:
this seems safe to me, since even if you added some kind of backend autocapture it would likely by framework specific and end up being e.g. rails not ruby autocapture

will leave team vibes to have the final say :)